tests/test_format.py

```diff
@@ -15,38 +15,6 @@
     all_data_cases,
 )
 
-PY310_CASES: List[str] = [
-    "starred_for_target",
-    "pattern_matching_simple",
-    "pattern_matching_complex",
-    "pattern_matching_extras",
-    "pattern_matching_style",
-    "pattern_matching_generic",
-    "parenthesized_context_managers",
-]
-
-PY311_CASES: List[str] = [
-    "pep_654",
-    "pep_654_style",
-]
-
-PREVIEW_CASES: List[str] = [
-    # string processing
-    "cantfit",
-    "comments7",
-    "comments8",
-    "long_strings",
-    "long_strings__edge_case",
-    "long_strings__regression",
-    "percent_precedence",
-    "remove_except_parens",
-    "remove_for_brackets",
-    "one_element_subscript",
-    "remove_await_parens",
-    "return_annotation_brackets",
-    "docstring_preview",
-]
-
 SOURCES: List[str] = [
```

Comment:
Should we also walk the source directories here and get all of the Python files?

Reply:
I believe that the right thing to do is to add the following step/job to the CI process:

```shell
pip install -e .
black --check src
```

**This should not be a unit test in my opinion**, but another step/job in the CI process.

If you want an example, see what I did for my tool [Statue](https://github.com/saroad2/statue) in the CI process.
In the end of the CI I added a step called "Run on self" that checks that the code of statue pass the static code analysis ran by statue.

Reply:
True, it appears we only check each file with default params, so that would work. I don't have a strong preference for it being in tests or CI. Perhaps [here](https://github.com/psf/black/blob/main/.github/workflows/test.yml#L53) before publishing test results, so that the whole version matrix is tested. Other opinions?